public/docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,337 @@
+# Quickstart
+
+This page will take you through the basics of creating a Truffle project and migrating (deploying) a smart contract to a blockchain.
+
+## Creating a project
+
+To use most Truffle commands, you need to run them against an existing Truffle project. So the first step is to create a Truffle project.
+
+You can create a bare project template, but for those just getting started, you can use [Truffle Boxes](/boxes), which are example applications and project templates. We'll use the [MetaCoin box](/boxes/metacoin), which creates an [ERC20-style token](https://en.wikipedia.org/wiki/ERC20):
```

Comment:
Did we actually upgrade the Metacoin box to be ERC20? I don't think we did.

Reply:
Oh, actually I don't know. I just assumed that MetaCoin was ERC20, as I didn't realize that one could have a token that wasn't ERC20. What should I say instead?